src/locale.js

```diff
@@ -1,669 +1,549 @@
-/* @flow */
+"use strict";
+exports.__esModule = true;
+exports.COUNTRY_LANGS = exports.LANG = exports.COUNTRY = void 0;
 /* eslint max-lines: 0 */
-
```

Comment:
Unfortunately the flow-to-ts tool removes most newline characters.  It's a tedious process but we've been trying to re-add them manually in our TypeScript conversions.

Reply:
Also, I think this file can be deleted since we have `locale.ts` now!